f12-opshin-audit-closeout-report/opshin-close-out-report.typ

```diff
@@ -0,0 +1,254 @@
+// Set the background image for the page
+#let image-background = image("../images/Background-Carbon-Anastasia-Labs-01.jpg", height: 100%)
+#set page(
+  background: image-background,
+  paper: "a4",
+  margin: (left: 20mm, right: 20mm, top: 40mm, bottom: 30mm),
+)
+
+// Set default text style
+#set text(15pt, font: "Barlow")
+
+#v(3cm) // Add vertical space
+
+// Center-align the logo
+#align(
+  center,
+)[#box(width: 75%, image("../images/Logo-Anastasia-Labs-V-Color02.png"))]
+
+#v(1cm)
+
+// Set text style for the report title
+#set text(20pt, fill: white)
+
+// Center-align the report title
+#align(center)[#strong[PROJECT CLOSE-OUT REPORT]]
+
+#v(5cm)
+
+// Set text style for project details
+#set text(13pt, fill: white)
+
+// Display project details
+#table(
+  columns: 2,
+  stroke: none,
+  [*Project Number*],
+  [1200175],
+  [*Project manager*],
+  [Philip Disarro],
+  [*Date Started*],
+  [21-Oct-2024],
+  [*Date Completed*],
+  [05-May-2025],
+)
+
+// Reset text style to default
+#set text(fill: luma(0%))
+
+// Configure the initial page layout
+#set page(
+  background: none,
+  header: [
+    // Place logo in the header
+    #place(
+      right,
+      dy: 12pt,
+    )[#box(image(height: 75%, "../images/Logo-Anastasia-Labs-V-Color01.png"))]
+    #line(length: 100%) // Add a line under the header
+  ],
+  header-ascent: 5%,
+  footer: [
+    #set text(11pt)
+    #line(length: 100%) // Add a line above the footer
+    #align(center)[*Anastasia Labs* \ Project Close-out Report]
+  ],
+  footer-descent: 20%,
+)
+
+// This command can be turned on to justify the text #set par(justify: true)
+
+#show link: underline
+#show outline.entry.where(level: 1): it => {
+  v(12pt, weak: true)
+  strong(it)
+}
+
+// Initialize page counter
+#counter(page).update(0)
+
+#set page(footer: [
+  #set text(11pt)
+  #line(length: 100%) // Add a line above the footer
+  #align(center)[*Anastasia Labs* \ Project Close-out Report]
+  #place(right, dy: -7pt)[#counter(page).display("1/1", both: true)]
+])
+#v(100pt)
+
+// Configure the outline depth and indentation
+#outline(depth: 2, indent: 1em)
+
+// Page break
+#pagebreak()
+#set terms(separator: [: ], hanging-indent: 40pt)
+#v(20pt)
+/ Project Name: Opshin Audit
+/ URL: #link(
+    "link",
+  )[https://projectcatalyst.io/funds/12/cardano-open-developers/opshin-audit]
+
+#v(10pt)
+= List of KPIs
+#v(10pt)
+== Challenge KPIs
+#v(10pt)
+- *Improve Smart Contract Security:* By conducting an in-depth audit of the OpShin
```

Comment:
We should not use "Title Casing" for these points (I won't be commenting it for all), making them bold should be more than enough.